src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2636,6 +2636,65 @@
       text: az staticwebapp functions show -n MyStaticAppName -g MyResourceGroup
 """
 
+helps['staticwebapp backends'] = """
+type: group
+short-summary: Link or unlink a prexisting backend with a static webapp. Also known as "Bring your own Backend."
```

Comment:
nit: do we actually refer to it anywhere as "Bring your own Backend"? I see "Bring your own APIs" in the doc https://learn.microsoft.com/en-us/azure/static-web-apps/apis-overview#api-options